spec/requests/items_requests_spec.rb

```diff
@@ -190,5 +190,36 @@
         end
       end
     end
+
+    describe 'GET #show' do
+      let!(:item_category) { create(:item_category, name: 'CURRENTCATEGORY') }
+      let!(:item) { create(:item, organization: organization, name: "ACTIVEITEM", item_category_id: item_category.id, distribution_quantity: 1, on_hand_recommended_quantity: 1, package_size: 3) }
```

Comment:
Two things, to avoid false positives:  1/   Set each numerical value on the item to a different value.    2/  use fairly large values (at least in the 100s).   Because there are other numerical values on the screen,  if you use small values for the item, there is a high probability that it would match in the totals.